apps/examples/testcase/ta_tc/device_management/utc/utc_dm_main.c

```diff
@@ -145,6 +146,12 @@ static void wifiAutoConnect()
 			printf("[AutoConnect]WiFi is not working. Test Canceled\n");
 			return 0;
 		}
+		else {
```

Comment:
Not quite, although I see I miss a return 1 at the end. In that case it is required.